### ISSUE TITLE
Not working on Android API 21

### ISSUE THREAD

**Issue by tajchert** [andrewgiang/SpritzerTextView#18]
Text is aligned to left no matter which letter is "red". Correct behaviour would be to align red letter to markers - work as this on any tested API version < 21.

**Comment by tajchert** [comment]
Problem is in the system - https://code.google.com/p/android-developer-preview/issues/detail?id=110

**Comment by aliafshar** [comment]
I get around it my explicitly setting the font.

mSingleText = (SpritzerTextView) stub.findViewById(R.id.singletext);
mSingleText.setTypeface(Typeface.create(Typeface.MONOSPACE, Typeface.NORMAL));

**Comment by tajchert** [comment]
Yeah, I have created pull request with setting Typeface.MONOSPACE from code. Just waiting for author to merge it.

**Comment by aliafshar** [comment]
Could probably do this without a monospace font too. Would certainly look nicer.

**Comment by eh3andark** [comment]
how to remove "red" character... This character has problem in utf-8 ((persian language))

![capture](https://cloud.githubusercontent.com/assets/24735133/23269385/a569cf20-fa05-11e6-8142-eaf1c01ecbd1.PNG)

**Comment by tajchert** [comment]
@eh3andark I don't think it is related to previous issue.

**Comment by eh3andark** [comment]
@tajchert The application made by the latest version of the this library

**Comment by tajchert** [comment]
@eh3andark but how your issue is related to "Not working on Android API 21" and my first post, solved by my pull request #19 ?

**Comment by eh3andark** [comment]
@tajchert
I'm sorry ...
